partner/partner-management-service/src/main/java/io/mosip/pms/oauth/client/service/impl/ClientManagementServiceImpl.java

```diff
@@ -594,6 +598,13 @@ public io.mosip.pms.oauth.client.dto.ClientDetail getClientDetails(String client
 		dto.setRedirectUris(convertStringToList(result.get().getRedirectUris()));
 		dto.setGrantTypes(convertStringToList(result.get().getGrantTypes()));
 		dto.setClientAuthMethods(convertStringToList(result.get().getClientAuthMethods()));
+		try {
+			Map<String,String> clientNameLangMap = objectMapper.readValue(result.get().getClientNameLang(), new TypeReference<Map<String, String>>() {
```

Comment:
can we have a null check on result.get().getClientNameLang(). It can be null for OIDC Client.

Reply:
added null check